src/containers/PipelineRuns/PipelineRuns.test.js

```diff
@@ -362,3 +363,44 @@ it('An invalid filter value is disallowed and reported', async () => {
     )
   ).toBeTruthy();
 });
+
+it('TaskTree handles rerun event in PipelineRuns page', async () => {
+  const mockTestStore = mockStore(testStore);
+  const match = {
+    params: {},
+    url: '/pipelineruns'
+  };
+
+  const onSelect = jest.fn();
+  const { queryByText, getByTestId, getByText } = renderWithRouter(
+    <Provider store={mockTestStore}>
+      <Route
+        path="/pipelineruns"
```

Comment:
use router utils (e.g. `paths.pipelineRuns.all()` and `urls.pipelineRuns.all()`) instead of hardcoding, makes it easier to keep track of the values used and to update them in future